lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -38,7 +39,10 @@ function collectImageElementInfo() {
     };
   }
 
-  const htmlImages = [...document.querySelectorAll('img')].map(element => {
+  const allElements = getElementsInDocument();
```

Comment:
Oh, just that we went from 2 qSA to 1. Before we had a sep query just for `document.querySelectorAll('img')`. But now, `allElements` is inclusive of all the img elements on the page. Filtering `allElements` is probably not much faster, but it's nice to combine the collection calls.